src/Base/Models/PortalLinkGenerator.php

```diff
@@ -42,6 +43,11 @@ public function generateBasePortalLink(): string
     private function createPortalSlug(): self
     {
         $portalURL = $this->pdcSettings->getPortalURL();
+
+        if (! wp_http_validate_url($portalURL) && $this->pdcSettings->useShowOn()) {
```

Comment:
Kunnen we de check `! wp_http_validate_url($portalURL)` niet weglaten?

Als de pdc feature "show-on" aan staat dan gaat de gewone portal url sowieso niet meer werken voor items die op een subsite getoond moeten worden.

Misschien moeten we de optie "Portaal URL" bij de pdc settings zelfs verbergen als de optie "Tonen op" is aangevinkt.